modules/autoscaling.md

```diff
@@ -96,12 +96,40 @@ Cumulus's autoscaling module has the following usage:
 cumulus autoscaling [diff|help|list|migrate|sync] <asset>
 {% endhighlight %}
 
-Autoscaling groups can be diffed, listed, and synced (migrate is covered in the [following section](#migration)). These three actions do the following:
+Autoscaling groups can be diffed, listed, and synced (migrate is covered in [migration](#migration)). These three actions do the following:
 
 * `diff` - Shows the differences between the local definition and the AWS autoscaling configuration. If `<asset>` is specified, Cumulus will diff only the autoscaling group with that name.
 * `list` - Lists the names of all the autoscaling groups defined in local configuration
 * `sync` - Syncs local configuration with AWS. If `<asset>` is specified, Cumulus will sync only the autoscaling group with that name.
 
+
+### Syncing Size
+
+Because autoscaling groups can change in size in response to alarms or scheduled actions, the size (min, max, and desired instances) of an autoscaling group is synced a little differently than the other configuration options. If the local configuration has any scheduled actions configured, we will set the min and max size of the autoscaling group to match the last scheduled action that would have ran. Desired is never update unless it is not in the min/max bounds. This behavior can be overridden by passing the `--autoscaling-force-size` argument with the sync command, which will use the configured size on the autoscaling group instead of the most recent scheduled action.
```

Comment:
`This behavior` is a little ambiguous, since at first I thought it referred to the "not updating desired`. Perhaps `Scheduled actions can be ignored by passing ...`